rbac/api/cross_access/view.py

```diff
@@ -107,6 +116,54 @@ def list(self, request, *args, **kwargs):
             return self.replace_user_id_with_info(result)
         return result
 
+    def partial_update(self, request, *args, **kwargs):
+        """Patch a cross-account request."""
```

Comment:
Is this the method for the admin in target account to approve/deny the request?

Reply:
I think this would be the best/easiest way, yep! Might be worth ensuring that only admins can set 'approved' in the status?